tests/zfs-tests/tests/functional/pam/pam_basic.ksh

```diff
@@ -0,0 +1,49 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+. $STF_SUITE/tests/functional/pam/utilities.kshlib
+
+mounted 0
+keystatus unavailable
+
+genconfig "homes=$TESTPOOL/pam"
+echo "testpass" | /usr/bin/pamtester pam_zfs_key_test ${username} open_session
```

Comment:
Do not use the full path to `pamtester`. It's not there on FreeBSD, and more to the point, commands used by the tests are symlinked in a constrained bin/ path. So just `pamtester` is appropriate.